include/bluetooth/gatt.h

```diff
@@ -382,7 +382,7 @@ ssize_t bt_gatt_attr_read_service(struct bt_conn *conn,
 #define BT_GATT_SERVICE(_attrs)						\
 {									\
 	.attrs = _attrs,						\
-	.attr_count = ARRAY_SIZE(_attrs),				\
+	.attr_count = (u16_t)ARRAY_SIZE(_attrs),			\
```

Comment:
Could you fix this for `BT_GATT_CCC`  as well?

Reply:
Is it required there?  cfg_len is of type size_t.

Reply:
Could we just change attr_count to be size_t? u16_t is not buying us anything in the bt_gatt_service struct, since there's just going to be padding there up to 32 bits.

Reply:
You guys decide - I will update the PR accordingly.
(Can we assume/trust that size_t is 32 bit in this case?)

Reply:
size_t should always be compatible with sizeof(), so it should be compatible with ARRAY_SIZE which uses sizeof()